pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
   "grpcio-tools >= 1.54.2, < 2",
   "protobuf >= 4.25.3, < 6",
   "frequenz-client-base[grpcio] >= 0.5.0, < 0.6.0",
+  "pandas == 2.2.2"
```

Comment:
Do we really want to make an API client depend on pandas? For me this sounds like it should be put in a higher level.

Reply:
No, not at all. It was just a first shot as I am unsure of the data structure in which we want to supply values. I am happy to move away from pandas. The best option may also be connected to data structure that we will use when exposing the states @cwasicki?